sdv/model.py

```diff
@@ -841,11 +667,159 @@ async def get(self) -> TypedDataPointResult[List[str]]:
             raise
 
     async def set(self, value: List[str]):
+        await self._set(value, self.__class__.__name__)
+
+    def create_broker_data_point(self, value: List[str]):
+        array = StringArray(values=value)
+        return BrokerDatapoint(string_array=array)
+
+
+class BatchSetBuilder:
+    """Collects data points to be updated in a single 'atomic' set"""
```

Comment:
Extend the documentation to describe how to be used. 
- apply is required to commit the update request

Reply:
Done